### DISCUSSION TITLE
Crazyflie still stay in DFU mode, after Flashing successfully

### DISCUSSION THREAD

**Discussion by LPKPK** [bitcraze/discussions#346] [Q&A]
I tried to flash the drone in PX4 support firmware, following the introduction from [PX4 user guild](https://docs.px4.io/main/en/complete_vehicles/crazyflie21.html). After flashing my crazyflie 2.1, everything shown on my terminal was successsful, but the crazyflie was still in DFU mode. 

1. Is that the firmware not support PX4 development now? I just bought a basic crazyflie without any bundle. 
2. If PX4 is not supported, how can I flash in crazyflie firmware without crazyradio? Any guildance with be helpful!

Pengkai

**Comment by knmcguire** [comment]
Hi! As the guide says, the support and guide are experimental for PX4. We hope to change that in the future but it won't be done for the next months.

First try these instructions to recover the crazyflie through DFU.: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/dfu/. Be very careful since you might flash over the bootloader. 

It would be better to have a crazyradio to be honest. The Px4 guide also ask for this
